data-es2016plus.js

```diff
@@ -7361,6 +7361,61 @@ exports.tests = [
       }
     ]
   },
+  {
+    name: 'Array Grouping',
+    category: '2024 features',
+    significance: 'small',
+    spec: 'https://github.com/tc39/proposal-array-grouping',
+    subtests: [
+      {
+        name: 'Object.groupBy()',
+        mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy',
+        exec: function () {/*
+          const array = [1, 2, 3, 4];
+          const obj = Object.groupBy(array, (num, index) => {
+            return num % 2 === 0 ? 'even': 'odd';
+          });
+          return obj['even'][0] == 2 && obj['odd'][0] == 1;
```

Comment:
```suggestion
          return !('toString' in obj) && obj.even[0] == 2 && obj.odd[0] == 1;
```
to ensure it's got a null prototype

Reply:
Done.